sdks/java/io/hadoop-input-format/pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0"?>
```

Comment:
this whole file looks like it's using tabs rather than spaces. Can we switch it over to 2 spaces?